README.md

```diff
@@ -1,38 +1,38 @@
-# Lolex [![Build Status](https://travis-ci.org/sinonjs/lolex.svg?branch=master)](https://travis-ci.org/sinonjs/lolex)
+# `@sinonjs/fake-clock` [![Build Status](https://travis-ci.org/sinonjs/fake-clock.svg?branch=master)](https://travis-ci.org/sinonjs/fake-clock)
 
-JavaScript implementation of the timer APIs; `setTimeout`, `clearTimeout`, `setImmediate`, `clearImmediate`, `setInterval`, `clearInterval`, `requestAnimationFrame`, `cancelAnimationFrame`, `requestIdleCallback`, and `cancelIdleCallback`, along with a clock instance that controls the flow of time. Lolex also provides a `Date` implementation that gets its time from the clock.
+JavaScript implementation of the timer APIs; `setTimeout`, `clearTimeout`, `setImmediate`, `clearImmediate`, `setInterval`, `clearInterval`, `requestAnimationFrame`, `cancelAnimationFrame`, `requestIdleCallback`, and `cancelIdleCallback`, along with a clock instance that controls the flow of time. FakeClock also provides a `Date` implementation that gets its time from the clock.
 
-In addition in browser environment lolex provides a `performance` implementation that gets its time from the clock. In Node environments lolex provides a `nextTick` implementation that is synchronized with the clock - and a `process.hrtime` shim that works with the clock.
+In addition in browser environment `@sinonjs/fake-clock` provides a `performance` implementation that gets its time from the clock. In Node environments FakeClock provides a `nextTick` implementation that is synchronized with the clock - and a `process.hrtime` shim that works with the clock.
 
-Lolex can be used to simulate passing time in automated tests and other
+`@sinonjs/fake-clock` can be used to simulate passing time in automated tests and other
 situations where you want the scheduling semantics, but don't want to actually
-wait (however, from version 2.0 lolex supports those of you who would like to wait too).
+wait (however, from version 2.0 `@sinonjs/fake-clock` supports those of you who would like to wait too).
```

Comment:
this is just weird, isn't it? There won't be a version 2 of this new package

Reply:
Yeah. Perhaps it should just be dropped